oceannavigator/__init__.py

```diff
@@ -63,6 +64,14 @@ def create_app(testing: bool = False, dataset_config_path: str = ""):
     app.config.from_pyfile("oceannavigator.cfg", silent=False)
     app.config.from_envvar("OCEANNAVIGATOR_SETTINGS", silent=True)
     app.testing = testing
+    app.git_hash = (
+        subprocess.check_output(["git", "rev-parse", "HEAD"]).decode("ascii").strip()
```

Comment:
I'd suggest extracting these two functions into `utils` maybe and catch any exceptions that may arise otherwise the app won't start.

```py

def subprocess_check_output(*args, **kwargs) -> str:
    try:
        return subprocess
            .check_output(*args, **kwargs, stderr=subprocess.STDOUT)
            .decode("ascii")
            .strip()
    except subprocess.CalledProcessError as e:
        print(f"Exception on process - args={args} rc=#{e.returncode} output=#{e.output})
        return ""
end
```

Reply:
Oh good catch. I can't put that function in `utils.misc` because it imports `DatasetConfig` from `oceannavigator` which will raise an error due to the circular imports but I add it below these calls in `oceannavigator.__init__` if that sounds good to you.